assets/js/Line.test.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { mount } from 'enzyme';
+
+import Line from './Line';
+
+test('does not show messages that have expired', () => {
```

Comment:
This doesn't seem like the sort of thing that should be tested here. I think it's being tested by the Sign, right? Rather, this should test something about the correct signs appearing for a given line.